dll/win32/browseui/CAutoComplete.cpp

```diff
@@ -272,17 +272,21 @@ EditWordBreakProcW(LPWSTR lpch, INT index, INT count, INT code)
         {
             if (index)
                 --index;
-            while (index && !IsWordBreak(lpch[index]))
+            while (index && 
+            (!IsWordBreak(lpch[index-1]) || (IsWordBreak(lpch[index]) && IsWordBreak(lpch[index-1])))
```

Comment:
indentation broken.
Also, since your while-loop condition is more than one line long, please put the body of the loop within curly brackets.